scripts/cassandra-integration-test.sh

```diff
@@ -53,13 +46,14 @@ apply_schema() {
 
 run_integration_test() {
   local version=$1
+  local major_version=${version%%.*}
```

Comment:
Can we do the same with es/os scripts? Their matrix was changed "7.x" -> "7", which makes the workflow names look weird. I would prefer to go back to 7.x / 8.x

Reply:
Can be a separate PR for that.

Reply:
ok i will create a new pr after this